### PR TITLE
docs: Update tool examples to latest configuration

### PR DESCRIPTION
**Pull request by MoritzWeber0**

### CODE REVIEW

**Comment by jamilraichouni** [comment]
I'd like to know, why the whole code listing block (starting with tripe backticks) is indented below that `=== "Capella/Papyrus/Eclipse"` heading. Why are lines 324 and following indented?

image

The rendered output shows, that there is something wrong. We have no syntax highlighting and the first line (see marker on the screenshot) should not be part of the code listing:

image

**Comment by jamilraichouni** [comment]
In Markdown I do not know the meaning of a `===` prefix which seems to have no rendering style effect:

`=== "Capella/Papyrus/Eclipse"`

**Review by jamilraichouni — CHANGES_REQUESTED** [review]

**Comment by MoritzWeber0** [comment]
> In Markdown I do not know the meaning of a `===` prefix which seems to have no rendering style effect:
> 
> `=== "Capella/Papyrus/Eclipse"`

We don't use Markdown, we use mkdocs-material for the documentation, which has more features. The rendered output looks good for me: https://capella-collab-manager--1605.org.readthedocs.build/1605/admin/tools/configuration/#configuration-examples

File: docs/docs/admin/tools/configuration.md [hunk 1]
```diff
@@ -337,26 +337,27 @@ which we provide as part of our
           limits: 6Gi
       environment:
         RMT_PASSWORD: "{CAPELLACOLLAB_SESSION_TOKEN}"
+        ECLIPSE_PROJECTS_TO_LOAD: "{CAPELLACOLLAB_SESSION_PROVISIONING}"
       connection:
         methods:
-          - identifier: guacamole
+          - id: guacamole
```
[the remark this answers is not in the document]
Comment (MoritzWeber0) [reply docs/docs/admin/tools/configuration.md:343]: Was a team decision back then to rename identifier to id. The decision has been made and that's how the configuration option is called now. This is just the PR to adjust the outdated documentation to the renamed attribute.

**Comment by jamilraichouni** [comment]
> > In Markdown I do not know the meaning of a `===` prefix which seems to have no rendering style effect:
> > `=== "Capella/Papyrus/Eclipse"`
> 
> We don't use Markdown, we use mkdocs-material for the documentation, which has more features. The rendered output looks good for me: https://capella-collab-manager--1605.org.readthedocs.build/1605/admin/tools/configuration/#configuration-examples

Ah, sorry. Have not used that so far and because of the file extension I thought this is supposed to be Markdown.

**Review by jamilraichouni — APPROVED** [review]
LGTM

**Pull request merged**